phpunit.xml.dist

```diff
@@ -14,4 +14,12 @@
             <directory suffix=".php">./tests/</directory>
         </testsuite>
     </testsuites>
+    <php>
+        <env name="DB_CONNECTION" value="mysql"/>
+        <env name="DB_USERNAME" value="root"/>
+        <env name="DB_PASSWORD" value="root"/>
+        <env name="DB_DATABASE" value="nova_pages_tool_testing"/>
```

Comment:
Je kan env vars ook in de workflow file meegeven. Dan hoef je dit niet allemaal hard coded erin te zetten en kan men lokaal `.env.testing` blijven gebruiken.

Reply:
Aannemende dat dat zo werk met Testbanche.

Reply:
Testbench gebruikt geen .env variabelen. Kan je wel instellen door een bepaalde property te zetten op de testcase, maar dan zoekt ie een .env bestand in de application path wat weer in een laravel vendor dir zit, super stom, lijkt een bug. Dan moeten we ook weer de env application path moeten definieren in de testcase.

Uiteindelijk las ik dat PHPunit adviseert om een .dist file te gebruiken die je in de repo houdt. Zodat je lokaal met een eigen `phpunit.xml` file deze weer kan overschrijven.